### ISSUE TITLE
"options must be an object" error

### ISSUE THREAD

**Issue by pak11273** [storyblok/gatsby-source-storyblok#169]
[storyblokurl]: https://www.storyblok.com?utm_source=github.com&utm_medium=issue-template

"options must be an object" error
---

## Expected Behavior

When starting my server I get this error which I haven't encountered before.  I removed node_modules and restarted the server and I've downgraded this plugin but it's still not working with several versions.  I've double checked my configuration but everything checks out I think. It should have  no errors and server should run like it has previously done before.
## Steps to Reproduce

1. gatsby develop

```javascript
// This is the error produced after running `gatsby develop`:

  16 |   });
  17 |
> 18 |   const space = await Sync.getSpace();
     |                            ^
  19 |   const languages = options.languages ? options.languages : space.language_codes;
  20 |   languages.push('');
  21 |

File: node_modules/gatsby-source-storyblok/gatsby-node.js:18:28
```

This is my gatsby.config.js file: 
```javascript
module.exports = {
	plugins: [
                ...
		{
			resolve: 'gatsby-source-storyblok',
			options: {
				accessToken: *********secret***********,
				version: 'draft',
				region: 'us'
			}
		}
         ]
}
```

And this is my `layout.js` file:

```javascript
// imports

const sbConfig = configuration.plugins.find((item) => item.resolve === 'gatsby-source-storyblok')

storyblokInit({
	// accessToken: **************secret**********,
	accessToken: sbConfig.options.accessToken,
	apiOptions: {
		region: 'us'
	},
	use: [apiPlugin],
	components: {
		slide: Slider,
		teaser: Teaser
        }
})

I'm not sure what's going on.  The application was running fine before, but as I've said this last attempt was run after  I deleted my `node_modules` folder and then reinstalled packages with `yarn`.  Thanks in advance!

**Comment by zjr** [comment]
We found the cause of the issue for us.

We had a dependency of Axios@v1.x which was incompatible with storyblok-js-client@v4.x (it doesn't specify an axios version) which is in turn a dependency of this package, gatsby-source-storyblok. We downgraded to axios@^0.27.2 and that resolved the issue for us.

It looks like storyblok-js-client dropped the axios dependency in v5 a couple months ago, so it seems that upgrading that dependency in this repo would resolve the issue for others who specify an axios dependency >= v1.
